CONTRIBUTORS.md

```diff
@@ -1,6 +1,5 @@
 Following is a list of contributors in alphabetical order:
 
-- Aaraj Habib
 - Ashwini Vaidya
```

Comment:
is this removal intentional?

Reply:
This was definitely not intentional. Perhaps I botched a merge fix somewhere? Thank you, I will add this person back to the list